SwiftParcel.Services.Orders/src/SwiftParcel.Services.Orders.Application/SwiftParcel.Services.Orders.Application/DTO/ParcelDto.cs

```diff
@@ -46,6 +52,12 @@ public ParcelDto(Parcel parcel)
             CreatedAt = parcel.InquireDate;
             ValidTo = parcel.ValidTo;
             CalculatedPrice = parcel.CalculatedPrice;
+            PriceBreakDown = parcel.PriceBreakDown.Select(x => new PriceBreakDownItemDto
```

Comment:
Also there I have to change this line to: "PriceBreakDown = parcel.PriceBreakDown?.Select(x => new PriceBreakDownItemDto" because during getting orders I noticed an error.
![problem2](https://github.com/SaintAngeLs/courier_app/assets/148250180/b71c57c4-699e-436e-b1c6-f50a2170b04f)